INaturalistIOS/INaturalistIOS-Info.plist

```diff
@@ -44,11 +44,11 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>180</string>
+	<string>182</string>
```

Comment:
Please don't commit changes to the Info.plist in pull requests, unless the change is related to a new feature or is required for all forks of the app, including the main iNaturalist store app.